dashboard/config/scripts/levels/CSD U3 AnimationsMulti introducing multiframe.level

```diff
@@ -133,10 +133,10 @@
       "console.log": null,
       "functionParams_none": null,
       "callMyFunction": null,
-      "comment": null
+      "comment": null,
+      "stopSound": null
     },
     "show_debug_watch": "false",
-    "encrypted_examples": "hBYEo3wudGBFMFzTZ/joDBvBc0h7k5p7JWdsCu9nmhxMPKlb7gdxA2UfwmPx\nbdwa\n",
```

Comment:
I don't know too well either, but I suspect deleting this key is not the desired behavior. I suspect it needs to be regenerated based on level contents. Not sure if this needs to be done on LB. Might ask around on slack to see who knows about these.